src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.component.js

```diff
@@ -207,6 +235,9 @@ const RegistrationDataEntryPlain = (
                                         selectedScopeId={selectedScopeId}
                                         open={modalIsOpen}
                                         onCancel={closeModal}
+                                        renderCardActions={({ item }) =>
+                                            <CardListButton teiId={item.id} orgUnitId={item.tei.orgUnit} item={item} />
+                                        }
```

Comment:
It might look odd at the beggining that we are passing the _exact_ same property in the `PossibleDuplicatesDialog` and the `DataEntryWidgetOutput`. 

I wanted to remind you though that the `DataEntryWidgetOutput` has its own modal, which pops up when you click the warning indicator from the right side.